app/models/foreman_openscap/policy.rb

```diff
@@ -168,8 +164,38 @@ def to_enc
       }.merge(period_enc)
     end
 
+    def should_validate?(step_i)
+      if new_record? && wizard_initiated?
+        step_index > step_i
+      elsif new_record? && !wizard_initiated?
+        true
+      else
+        !id.blank?
+      end
+    end
+
+    def wizard_initiated?
+      @wizard_initiated
+    end
+
+    def update_attributes(hash)
```

Comment:
I'd recommend moving this to before_validation callback method for following reasons:
* this overrides internal #updates_attributes AR method, if they later change the signature of this method it will break
* if we reset in before_validation we get closer to point when the record is normally validates